packages/jest-cli/src/cli/index.js

```diff
@@ -40,9 +41,9 @@ export async function run(maybeArgv?: Argv, project?: Path) {
     const {results, globalConfig} = await runCLI(argv, projects);
     readResultsAndExit(results, globalConfig);
   } catch (error) {
-    clearLine(process.stderr);
-    clearLine(process.stdout);
-    console.error(chalk.red(error.stack));
+    clearLine(output.error);
```

Comment:
can we flip this? so you say `output.clearLine(output.error, output.log)`

Reply:
Great idea, flipped it around 🙂